packages/truffle-contract/lib/utils.js

```diff
@@ -149,7 +149,7 @@ const Utils = {
 
     const expected_arg_count = methodABI ? methodABI.inputs.length : 0;
 
-    tx_params = {};
+    const tx_params = {};
```

Comment:
Hmm we can't do this because it is redefined on line 159. We should make this a `let`.

Reply:
I just found out, yes. I'll fix it